HDF5Examples/C/TUTR/CMakeLists.txt

```diff
@@ -0,0 +1,113 @@
+cmake_minimum_required (VERSION 3.18)
+project (HDF5Examples_C_TUTR C)
+
+#-----------------------------------------------------------------------------
+# Define Sources
+#-----------------------------------------------------------------------------
+include (C_sourcefiles.cmake)
+
+foreach (example_name ${examples})
+  add_executable (${EXAMPLE_VARNAME}_tutr_${example_name} ${PROJECT_SOURCE_DIR}/${example_name}.c)
+  target_compile_options(${EXAMPLE_VARNAME}_tutr_${example_name}
+      PRIVATE
+          "$<$<BOOL:${${EXAMPLE_VARNAME}_USE_16_API}>:-DH5_USE_16_API>"
+          "$<$<BOOL:${${EXAMPLE_VARNAME}_USE_18_API}>:-DH5_USE_18_API>"
+          "$<$<BOOL:${${EXAMPLE_VARNAME}_USE_110_API}>:-DH5_USE_110_API>"
+          "$<$<BOOL:${${EXAMPLE_VARNAME}_USE_112_API}>:-DH5_USE_112_API>"
+          "$<$<BOOL:${${EXAMPLE_VARNAME}_USE_114_API}>:-DH5_USE_114_API>"
+          "$<$<BOOL:${${EXAMPLE_VARNAME}_USE_116_API}>:-DH5_USE_116_API>"
```

Comment:
Should remove this line for the 1.14 branch

Reply:
Doesn't matter much, they weren't removed last time. Besides. as a self-contained project it could be used with any library version. 
I don't think it is worth the trouble.

Reply:
Not difficult to do - removed 24 such lines in 15 CMakeLists.txt under HDF5Examples